src/layouts/EditNav.jsx

```diff
@@ -96,21 +98,27 @@ export default class EditTree extends Component {
       tree, parentId: source.parentId, index: source.index,
     }).data.type;
 
-    const destinationParentType = this.getParentFromTreeposition({
+    const destinationParent = this.getParentFromTreeposition({
```

Comment:
in `sourceParentType` right above this line of code, remove `index: source.index` since `index` is not an argument to `getParentFromTreeposition`

Reply:
oh yeah good point